src/briefcase/integrations/subprocess.py

```diff
@@ -86,6 +87,24 @@ def final_kwargs(self, **kwargs):
         if not ('text' in kwargs or 'universal_newlines' in kwargs):
             kwargs['text'] = True
 
+        # For Windows, convert start_new_session=True to creation flags
+        if self.command.sys.platform == "win32":
+            try:
+                if kwargs.pop('start_new_session') is True:
+                    # CREATE_NEW_PROCESS_GROUP: Makes the new process the root process
+                    #     of a new process group. This also disables CTRL+C signal handlers
+                    #     for all processes of the new process group.
+                    # CREATE_NO_WINDOW: Creates a new console for the process but does not
+                    #     open a visible window for that console. This flag is used instead
+                    #     of DETACHED_PROCESS since the new process can spawn a new console
+                    #     itself (in the absence of one being available) but that console
+                    #     creation will also spawn a visible console window.
+                    new_session_flags = self._subprocess.CREATE_NEW_PROCESS_GROUP | self._subprocess.CREATE_NO_WINDOW
+                    # merge these flags in to any existing flags already provided
+                    kwargs['creationflags'] = kwargs.get('creationflags', 0) | new_session_flags
```

Comment:
This could have some interesting behaviours if the user provides their own creationflags - e.g., if they provide DETACHED_PROCESS, they're going to get creation flags with both DETACHED_PROCESS *and* CREATE_NO_WINDOW. It might be worth raising the "you've also specified creationflags" as an error in this case - at least for now, it won't be a problem because we're not using any specialist creationflags; but if this ever changes in the future, the exception will make sure the problem is caught and doesn't cause any hard-to-diagnose errors.

Reply:
Added a warning....let me know if it might go a little overboard :)

Reply:
I'd lean towards `AssertionError` here rather than a bare `Exception`. Definitely doesn't need a custom Exception class, though.

Reply:
Updated to `AssertionError`.